meilisearch/client.py

```diff
@@ -14,7 +14,7 @@ class Client():
     config = None
     http = None
 
-    def __init__(self, url, apiKey=None):
+    def __init__(self, url, apiKey=None, timeout=10):
```

Comment:
was `10` the previous default timeout?

Reply:
I can't find any documentation on the default timeout for `requests`. The default for `httpx` is 5, requests seems to be longer so I picked 10. Looking through the `requests` code base it seems as though the default may be `None` so I could change the 10 to this if you like.

Reply:
I see what you mean on the setting of the timeout test now. I will add these.

Reply:
Fiou oke I found it haha! Seems like the default value is `None`
```python
 def request(self, method, url,
            params=None, data=None, headers=None, cookies=None, files=None,
            auth=None, timeout=None, allow_redirects=True, proxies=None,
            hooks=None, stream=None, verify=None, cert=None, json=None):
```

Here is the [doc](https://requests.readthedocs.io/en/latest/_modules/requests/sessions/#Session.request)